source/_components/recorder.markdown

```diff
@@ -102,6 +104,27 @@ If you only want to hide events from e.g. your history, take a look at the [`his
 
 Not all Python bindings for the chosen database engine can be installed directly. This section contains additional details which should help you to get it working.
 
+### {% linkable_title MariaDB %}
```

Comment:
As MariaDB is drop-in replacement for MySQL I would like to suggest to keep only one section.

Reply:
The problem is that it isn't since the dependencies are different. Sure we could only have one section but it would be harder to search for in that case and be a tad more convoluted for a new user.

Reply:
It's `libmariadbclient-dev` vs. `libmysqlclient-dev`.